src/Libraries/CoreNodesUI/Input/BoolSelector.cs

```diff
@@ -23,6 +23,17 @@ protected override bool DeserializeValue(string val)
             }
         }
 
+        protected override bool UpdateValueCore(string name, string value)
+        {
+            if (name == "Value")
+            {
+                Value = DeserializeValue(value);
+                return true;
+            }
+
+            return base.UpdateValueCore(name, value);
+        }
+
```

Comment:
@Benglin, yes, it's being called only from DynamoWebServer when Flood needs to change boolean node's value. These changes were added by me [here](https://github.com/Benglin/Dynamo/pull/231) but after big merge I don't see them anymore